.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  extends: "steelbrain"
```

Comment:
Then you should specify that as a dev dependency, as it stands this `.eslintrc` means nothing...